### ISSUE TITLE
How to fine tune Tesseract 4.1.1 with make? (I've searched in many places, nothing else worked)

### ISSUE THREAD

**Issue by ryancarloscorrea** [tesseract-ocr/tesseract#3104]
Hello, I need to adjust the tesseract to improve the accuracy for my documents (Brazilian id's), I have 694.png and 694.gt.txt. I did the "make training", it generates me a .treineddata file, but it doesn't help me much because the hit rate with my documents is very low. I read that in these cases it would be better to fine tune, but I couldn't do it with make, I only saw examples with .sh, with make I didn't find anything that worked. I need a way to increase the hit rate urgently, can someone help me? What should I do in this case?

**Comment by Shreeshrii** [comment]
Use the tesstrain repo which has make based training.

**Comment by ryancarloscorrea** [comment]
I already used it, but the generated model is very bad alone, I would like to add this model to por.treineddata. How can I do this?

**Comment by Shreeshrii** [comment]
Did you use a start_model and limit the number of iterations?

**Comment by ryancarloscorrea** [comment]
I believe so ... in the Makefile: ifdef START_MODEL.
I have limited the number of iterations, I have tried several values ...

**Comment by ryancarloscorrea** [comment]
Is there a Makefile ready to do finetune? I'm not getting at all...
Is Finetune the best option for me?

**Comment by Shreeshrii** [comment]
Please post tesstrain make related queries in tesstrain repo. You should share your training data in order to get any meaningful help.

**Comment by ryancarloscorrea** [comment]
Here is the data .png, .box, .lstmf, .gt.txt
The Makefile
the .traineddata

I believe I should do fine tuning, but I don't know how to do that ...

[data-training.zip](https://github.com/tesseract-ocr/tesseract/files/5262412/data-training.zip)

**Comment by ryancarloscorrea** [comment]
I think I figured out what I need to do. I need to add my training data to "por.traineddata (portuguese)", how can I do this from make? I've seen the google group, forums, but nothing works for me ...

**Comment by Shreeshrii** [comment]
I think the problem is the quality of your images. If you fix that you will get better results with the official por.traineddata also.

See

https://tesseract-ocr.github.io/tessdoc/ImproveQuality.html

https://stackoverflow.com/questions/9480013/image-processing-to-improve-tesseract-ocr-accuracy
